### PR TITLE
ci(SFINT-2577): Ensure Prettier run properly on UNIX.

### PR DESCRIPTION
**Pull request by louis-bompart**
The glob was not resolved the same way due to escaping on bash vs Powershell, this is now fixed.
Also now using the standard .prettierrc.js, because .json is not standard.

Bash were the ones with degradation, and thus the CI did not trip on linting ('cause he's on Ubuntu)

### CODE REVIEW

**Comment by louis-bompart** [comment]
19acba5 failed as expected.
Now linting the files.

File: .prettierignore [hunk 1]
```diff
@@ -1,10 +1,10 @@
-*.svg
```
Comment (Neilerino) [inline .prettierignore]: this is probably a silly question but what is different here? The changes look the same to me

Comment (louis-bompart) [reply .prettierignore]: No such thing has a silly question ;P 
It's the end of line terminator that have changed: it was a CRLF, it's now a LF, as specified by the .prettierrc.js

Comment (Neilerino) [reply .prettierignore]: Gotcha! Thanks Louis!

File: package.json [hunk 2]
```diff
@@ -23,8 +23,8 @@
         "css": "node ./scripts/css.js",
         "dev": "node ./scripts/dev.server.js",
         "watchTest": "karma start ./config/karma.config.js",
-        "lint": "prettier --check **/*",
-        "lint-fix": "prettier --write **/* ",
+        "lint": "prettier --check \"./**/*\" --config \".prettierrc.js\" --ignore-path \".prettierignore\"",
```
Comment (jeremierobert-coveo) [inline package.json]: Why not targetting the json instead ?

Comment (louis-bompart) [reply package.json]: Because I changed it to a .js to be consistent with the other config files

Comment (louis-bompart) [reply package.json]: Because I changed it to a .js to be consistent with the other config files

Comment (jeremierobert-coveo) [reply package.json]: So where is you change to enable linux ?

Comment (louis-bompart) [reply package.json]: It's the wrapping of the glob (so `**/*` to `\"./**/*\"`)

Comment (louis-bompart) [reply package.json]: It's the wrapping of the glob (so `**/*` to `\"./**/*\"`)

**Pull request merged**